redfish-core/lib/ethernet.hpp

```diff
@@ -2073,68 +2073,201 @@ inline void afterVlanCreate(
                             BMCWEB_REDFISH_MANAGER_URI_NAME, vlanInterface);
     asyncResp->res.addHeader("Location", vlanInterfaceUri.buffer());
 }
+inline void handleNetworkPortPatch(
+    App& app, const crow::Request& req,
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+    const std::string& managerId, const std::string& portId)
+{
+    if (!redfish::setUpRedfishRoute(app, req, asyncResp))
+    {
+        return;
+    }
+    if (managerId != BMCWEB_REDFISH_MANAGER_URI_NAME)
+    {
+        messages::resourceNotFound(asyncResp->res, "Manager", managerId);
+        return;
+    }
+    std::optional<bool> emitLldp;
+    if (!json_util::readJsonPatch(req, asyncResp->res, "Ethernet/LLDPEnabled",
+                                  emitLldp))
+    {
+        return;
+    }
+    if (emitLldp)
+    {
+        setDbusProperty(
+            asyncResp, "xyz.openbmc_project.Network",
+            sdbusplus::message::object_path("/xyz/openbmc_project/network") /
+                portId,
+            "xyz.openbmc_project.Network.EthernetInterface", "EmitLLDP",
+            "LLDPEnabled", *emitLldp);
+    }
+}
+inline void handleNetworkPortGet(
+    App& app, const crow::Request& req,
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+    const std::string& managerId, const std::string& portId)
+{
+    if (!redfish::setUpRedfishRoute(app, req, asyncResp))
+    {
+        return;
+    }
+    if (managerId != BMCWEB_REDFISH_MANAGER_URI_NAME)
+    {
+        messages::resourceNotFound(asyncResp->res, "Manager", managerId);
+        return;
+    }
+    std::string path = std::format("/xyz/openbmc_project/network/{}", portId);
```

Comment:
How does this handle

/redfish/v1/Managers/bmc/DedicatedNetworkPorts/abadnetworkportid

Check out https://github.com/openbmc/bmcweb/blob/60e995cd6ab15aac32e146860c2a99c069ea111f/COMMON_ERRORS.md#11-not-responding-to-404